packages/sandbox/API.md

```diff
@@ -25,6 +25,7 @@ export type Sandbox = {
 // @public (undocumented)
 export type SandboxDeleteOptions = {
     identifier?: string;
+    profile?: string;

```

Comment:
surprisingly `SandboxOptions` below already have `profile` but was unused.

Reply:
Looks like on our docs both `sandbox` and `sandbox delete` are mentioned to have `profile` which leads to confusion

Reply:
Which is fine, they both are supposed to handle that (except edge cases I described in PR description).
I was just surprised that `SandboxOptions.profile` already exists but noting in the code was reading (using) it to do anything useful.